### PR TITLE
Account Guides

### PR DESCRIPTION
**Pull request by dantheman999301**
Accessible at - https://docs.wealthkernel.com/docs/guides/branches/AccountGuides/docs/Welcome.md

### CODE REVIEW

File: docs/accounts/Basics.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+tags: [Accounts]
+---
+
+# Account Basics
+
+Accounts are an entity which hold information about the type product a client may wish to open, and validate that one of more parties can open a product of that type. Each account can have multiple parties and also have multiple Portfolios.
```
Comment (niall-maloney) [inline docs/accounts/Basics.md:7]:
```suggestion
Accounts are an entity which hold information about the type product a client may wish to open, and validate that one or more parties can open a product of that type. Each account can have both multiple Parties and multiple Portfolios associated to it.
```

File: docs/accounts/Basics.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+> SIPPs are currently in a private beta. If you wish to access them, please contact your account manager for more details
+
+## Account Lifecycle
+
+![Account-Status-SD](../../assets/images/accounts/Status-SD.png)
+
+| Status | Explanation |
+|---|---|
+| Pending | A very short lived state, before the account has checked whether all the relevant checks for parties have passed, such as KYC. |
+| Active | All checks have passed and the account is available to be used |
```
Comment (niall-maloney) [inline docs/accounts/Basics.md:36]:
```suggestion
| Active | All checks have passed and the account is available to be used. |
```

File: docs/accounts/Opening-Requirements.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+tags: [Accounts]
+---
+
+# Opening Requirements
+
+Each Account has it's own set of requirements that must be validated before an account can be opened. The list below explains what these are.
```
Comment (niall-maloney) [inline docs/accounts/Opening-Requirements.md:7]:
```suggestion
Each Account has its own set of requirements, listed below, that must be validated before the account can be opened.
```

File: docs/accounts/Basics.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+tags: [Accounts]
+---
+
+# Account Basics
+
+Accounts are an entity which hold information about the type product a client may wish to open, and validate that one or more parties can open a product of that type. Each account can have both multiple Parties and multiple Portfolios associated to it.
```
Comment (ZackJaime) [inline docs/accounts/Basics.md:7]: `....may wish to open, and validate that one or more parties....`

This transition doesn't quite scan to me, is this saying that an Account "validates that one or more parties can open a product of that type" ? And does that translates in practical terms to kyc checks?

Comment (ZackJaime) [reply docs/accounts/Basics.md:7]: Based on the assumptions in my previous comment, I ran it through:
https://hemingwayapp.com/
and got
```
Accounts hold information about the type of product a client may wish to open. It also validates that one or more parties can open a product of that type. Each account can have both many Parties and many Portfolios associated to it. 
```

Comment (dantheman999301) [reply docs/accounts/Basics.md:7]: I'll try and reword it 👍🏻

**Review by ZackJaime — APPROVED** [review]

**Pull request merged**